examples/gui_auto.py

```diff
@@ -4,14 +4,14 @@
 
 # test_example = true
 
-from wgpu.gui.auto import WgpuCanvas, run
+from rendercanvas.auto import RenderCanvas, loop
 
 try:
     from .triangle import setup_drawing_sync
 except ImportError:
     from triangle import setup_drawing_sync
 
-canvas = WgpuCanvas(title=f"Triangle example on {WgpuCanvas.__name__}")
+canvas = RenderCanvas(title=f"Triangle example on {RenderCanvas.__name__}")
```

Comment:
Rendercanvas has a little placeholder DSL for the title:
```suggestion
canvas = RenderCanvas(title=f"Triangle example on $backend")
```